.github/workflows/crowdin-update-resources.yml

```diff
@@ -0,0 +1,27 @@
+# Updates Crowdin with any changes in the English translation file (/assets/locales/en/translation.json)
+
+name: Update Crowdin English Resources
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
how do you think, would it make sense to also add a possibility of manual execution to this workflow, so one could push resources from a branch?

Reply:
I think the sync translations job only pulls non-English translations, so if Crowdin gets manually pushed resources that aren't on main, then the next sync translations job would only add the translated strings for all languages except English. Not sure how the app would react there.